tstest/integration/integration_test.go

```diff
@@ -268,73 +268,185 @@ func TestStateSavedOnStart(t *testing.T) {
 	d1.MustCleanShutdown(t)
 }
 
+// This handler looks for /auth/ URLs in the stdout from "tailscale up",
```

Comment:
`s/This handler/authURLHandler/`

This should also document that it counts machine auth urls as well, and that in both cases it fails the test if more than 1 is seen.